src/irmin-pack/checks.ml

```diff
@@ -176,6 +176,34 @@ struct
         const (fun root output () -> run ~root ~output) $ path $ dest)
   end
 
+  module Integrity_check = struct
+    module Store = Ext.Make (Conf) (M) (C) (P) (B) (H) (Node) (Commit)
+
+    let conf root = Config.v ~readonly:false ~fresh:false root
+
+    let auto_repair =
+      let open Cmdliner.Arg in
+      value
+      & flag
+        @@ info ~doc:"Automatically repair issues" ~docv:"AUTOREPAIR"
```

Comment:
```suggestion
        @@ info ~doc:"Automatically repair issues"
```

I believe this has no effect for flags. (c.f. https://erratique.ch/software/cmdliner/doc/Cmdliner.Arg.html)